release/serve_tests/workloads/single_deployment_1k_noop_replica.py

```diff
@@ -0,0 +1,196 @@
+            "-t",
+            str(NUM_CPU_PER_NODE),
+            "-d",
+            trail_length,
+            "http://127.0.0.1:8000/echo",
+        ],
+        stdout=PIPE,
+        stderr=PIPE,
+    )
+    proc.wait()
+    out, err = proc.communicate()
+    print(out.decode())
+    print(err.decode())
+
+    return out.decode()
+
+
+def shutdown_cluster():
+    pass
+
+
+@click.command()
+@click.option("--num-replicas", type=int)
+@click.option("--num-trials", type=int)
+@click.option("--trial-length", type=str)
+@click.option("--max-batch-size", type=int, default=DEFAULT_MAX_BATCH_SIZE)
+@click.option("--run-locally", type=bool, default=True)
+@click.option("--smoke-test", type=bool, default=True)
+def main(num_replicas: Optional[int], num_trials: Optional[int],
+         trial_length: Optional[str], max_batch_size: Optional[int],
+         run_locally: Optional[bool], smoke_test: Optional[bool]):
+
+    # Give default cluster parameter values based on smoke_test config
+    # if user provided values explicitly, use them instead.
+    if not smoke_test:
+        num_replicas = num_replicas or DEFAULT_FULL_TEST_NUM_REPLICA
+        num_trials = num_trials or DEFAULT_FULL_TEST_NUM_TRIALS
+        trial_length = trial_length or DEFAULT_FULL_TEST_TRAIL_LENGTH
+        print(f"\nRunning full test with {num_replicas} replicas, "
+              f"{num_trials} trails that lasts {trial_length} each.. \n")
+    else:
+        num_replicas = num_replicas or DEFAULT_SMOKE_TEST_NUM_REPLICA
+        num_trials = num_trials or DEFAULT_SMOKE_TEST_NUM_TRIALS
+        trial_length = trial_length or DEFAULT_SMOKE_TEST_TRAIL_LENGTH
+        print(f"\nRunning smoke test with {num_replicas} replicas, "
+              f"{num_trials} trails that lasts {trial_length} each.. \n")
+
+    # Choose cluster setup based on user config. Local test uses Cluster()
+    # to mock actors that requires # of nodes to be specified, but ray
+    # client doesn't need to
+    if run_locally:
+        num_nodes = int(math.ceil(num_replicas / NUM_CPU_PER_NODE))
+        print(f"\nSetting up local ray cluster with {num_nodes} nodes ....\n")
+        setup_local_single_node_cluster(num_nodes)
+    else:
+        print("\nSetting up anyscale ray cluster .. \n")
+        setup_anyscale_cluster()
+
+    print(f"\nDeploying with {num_replicas} target replicas ....\n")
+    deploy_replicas(num_replicas, max_batch_size)
+
+    print("\nWarming up cluster ....\n")
+    warm_up_cluster(5)
+
+    final_result = []
+    for iteration in range(num_trials):
+        print(f"\nStarting wrk trail # {iteration + 1} ....\n")
+        # For detailed discussion, see https://github.com/wg/wrk/issues/205
+        # TODO:(jiaodong) What's the best number to use here ?
+        num_connections = int(num_replicas * DEFAULT_MAX_BATCH_SIZE * 0.75)
+        decoded_out = run_one_trial(trial_length, num_connections)
+        metrics_dict = parse_wrk_decoded_stdout(decoded_out)
```

Comment:
print this each iteration?

Reply:
we already had this from wrk and prints from run_one_trial(), it formats way better than our json output